dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtility.java

```diff
@@ -289,34 +296,6 @@ public static boolean shouldSkipBomMember(Artifact artifact) {
     return false;
   }
 
-  /**
-   * Sets {@link #mavenRepositories} to search for dependencies.
-   *
-   * @param addMavenCentral if true, add Maven Central to the end of the repository list
-   * @throws IllegalArgumentException if a URL is malformed or not having an allowed scheme
-   */
-  public static void setRepositories(
-      Iterable<String> mavenRepositoryUrls, boolean addMavenCentral) {
-    ImmutableList.Builder<RemoteRepository> repositoryListBuilder = ImmutableList.builder();
-    for (String mavenRepositoryUrl : mavenRepositoryUrls) {
-      RemoteRepository repository = mavenRepositoryFromUrl(mavenRepositoryUrl);
-      repositoryListBuilder.add(repository);
-    }
-
-    if (addMavenCentral) {
-      repositoryListBuilder.add(CENTRAL);
-    }
-
-    mavenRepositories = repositoryListBuilder.build();
-  }
-
-  /** Adds {@link #mavenRepositories} to {@code collectRequest}. */
-  public static void addRepositoriesToRequest(CollectRequest collectRequest) {
-    for (RemoteRepository repository : mavenRepositories) {
-      collectRequest.addRepository(repository);
-    }
-  }
```

Comment:
Now we don't need to have these methods that use `mavenRepositories`, a mutable global variable.